tuplex/core/include/physical/PhysicalPlan.h

```diff
@@ -72,7 +72,8 @@ namespace tuplex {
         void executeWithParts(const tuplex::PhysicalPlan::File2FilePipeline &pip);
         double aggregateSamplingTime() const;
     public:
-
+        size_t getNumStages() const {return _num_stages;}
```

Comment:
can we add a doxygen comment specifying what number this returns? I.e, number of stages in a physical plan?

Reply:
Done.